### DISCUSSION TITLE
Does RustDesk record system audio when recording the session?

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#6895] [Q&A]
In the RustDesk session recording option, is it possible to also record the system audio, including the microphone? I did some tests and was unable to record the audio (no sound), only the screen is recorded. On both sides the audio option is activated. I can hear the audio from the remote computer during the session, but the recording is muted. I would like to be sure if this option does not actually exist, or if there is some other step to activate audio recording. I'm using version 1.2.4 (nightly) on Windows 11.

**Comment by onibg** [comment]
+1
I can hear the sound, but I'm unable to record it by clicking the recording button on RustDesk.
By the way, screen recording works perfectly.
Could you add an audio recording feature?
I'm using version 1.3.1(nightly) on Windows 10 21H2(19044.4780).

**Reply by Arthur2012** [reply]
+1
Remote sound is ok on the RustDesk, but no sound can be recorded by RustDesk native recorder.
The remote sound can be recorded by ShareX upon RustDesk, so it's RustDesk recorder issue, not remote/local audio driver issue.
Whatever the recording direction Win2Win or MAC2Win or Win2MAC, always only video track, no audio track.
<img width="521" height="654" alt="20260903_233406" src="https://github.com/user-attachments/assets/792b1b9a-796e-4865-9580-fcd7e0f372a9" />
